release-notes.md

```diff
@@ -0,0 +1,70 @@
+# Release Notes
+
+## 2.0
+
+Medic-conf v2.0.x contains breaking interface changes for the command-line and for the [declarative configuration system](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md).
+
+### Breaking Changes to Command-line Arguments
+Version 2.0 replaces our custom code for command-line argument parsing with [minimist](https://github.com/substack/minimist). This increases flexibility and more flexible ordering, but includes an interface change for a few commands.
+
+#### Commands Requiring '='
+The command
+  
+    medic-conf --url http://admin:pass@localhost:5988
+  
+now includes an `=` sign after the `--url`
+
+    medic-conf --url=http://admin:pass@localhost:5988
+    
+This affects the `url`, `instance`, `user`, and `shell-completion` arguments. Refer to 
+
+#### Action List Requires '--' Prefix
+The command
+
+    medic-conf --local convert-app-forms upload-app-forms my_form_1 my_form_2
+    
+now requires a '--' to separate actions from the list of forms:
+
+    medic-conf --local convert-app-forms upload-app-forms -- my_form_1 my_form_2
+
+### Breaking Changes to Declarative Configuration
+
+ The [declarative configuration system documentation](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md) contains up-to-date documentation of all interfaces.
+ 
+ #### Using Node Modules
```

Comment:
Minor nitpick, since it doesn't get picked up by GitHub as a header in the file diff
```suggestion
#### Using Node Modules
```